ipfx/dataset/mies_nwb_data.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, Any, List, Optional
+import abc
+
+from ipfx.stimulus import StimulusOntology
+from ipfx.dataset.lab_notebook_reader import LabNotebookReader
+from ipfx.dataset.ephys_nwb_data import EphysNWBData, get_finite_or_none
+
+
+class MIESNWBData(EphysNWBData):
+    """
+    Provides an Ephys Data Interface to a MIES generated NWB file
+
+    """
+
+    def __init__(self,
+                 nwb_file: str,
+                 lab_notebook_reader: LabNotebookReader,
```

Comment:
To me it makes more sense to construct labnotebook reader inside the constructor rather than ingesting it. The notebook is created from the same nwb_file as the MIESNWBData object. This prevents someone from injecting a wrong notebook from a different nwb file in the object. To me it seems that the notebook and MIESNWBData should live and die together.